content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,55 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful real-time data synchronization feature designed to facilitate seamless collaboration and data consistency across distributed systems. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated instantly and reliably to all connected clients in real-time. By providing bidirectional synchronization capabilities, LiveSync enables developers to effortlessly synchronize data between clients and servers, as well as between different clients, regardless of their location or network conditions.
```

Comment:
"seamless collaboration and data consistency across distributed systems" - our primary use case is really for synchronising changes from backend databases to frontend clients

Reply:
"instantly" sounds a bit like an over-promise to me, nothing can be instant :)

Reply:
"bidirectional synchronization capabilities" - livesync itself doesn't handle syncing writes from the client to the database, that must be handled by the developer through their existing write paths

Reply:
"as well as between different clients" - this sounds like livesync is peer-to-peer, which it isn't

Reply:
"regardless of their location or network conditions" - not quite true, if the client has no connectivity there's obviously nothing we can do about that, and we don't provide offline support. What we do do is provide connection recovery in case of an intermittent connectivity failure